src/homepageExperience/containers/CliWizard.tsx

```diff
@@ -0,0 +1,222 @@
+    finalFeedback: null,
+  }
+
+  private handleSelectBucket = (bucketName: string) => {
+    this.setState({selectedBucket: bucketName})
+  }
+
+  private handleMarkStepAsCompleted = () => {
+    this.setState({finishStepCompleted: true})
+  }
+
+  private setTokenValue = (tokenValue: string) => {
+    this.setState({tokenValue: tokenValue})
+  }
+
+  private setFinalFeedback = (feedbackValue: number) => {
+    this.setState({finalFeedback: feedbackValue})
+  }
+
+  handleNextClick = () => {
+    this.setState(
+      {
+        currentStep: Math.min(
+          this.state.currentStep + 1,
+          HOMEPAGE_NAVIGATION_STEPS_CLI.length
+        ),
+      },
+      () => {
+        event(
+          'firstMile.cliWizard.next.clicked',
+          {},
+          {
+            clickedButtonAtStep: this.state.currentStep - 1,
+            currentStep: this.state.currentStep,
+          }
+        )
+      }
+    )
+  }
+
+  handlePreviousClick = () => {
+    this.setState(
+      {currentStep: Math.max(this.state.currentStep - 1, 1)},
+      () => {
+        event(
+          'firstMile.cliWizard.previous.clicked',
+          {},
+          {
+            clickedButtonAtStep: this.state.currentStep + 1,
+            currentStep: this.state.currentStep,
+          }
+        )
+      }
+    )
+  }
+
+  handleNavClick = (clickedStep: number) => {
+    this.setState({currentStep: clickedStep})
+  }
+
+  renderStep = () => {
+    switch (this.state.currentStep) {
+      case 1: {
+        return <Overview wizard="cliWizard" />
+      }
+      case 2: {
+        return <InstallDependencies />
+      }
+      case 3: {
+        return (
+          <InitializeClient
+            wizardEventName="cliWizard"
+            setTokenValue={this.setTokenValue}
+            tokenValue={this.state.tokenValue}
+            onSelectBucket={this.handleSelectBucket}
+          />
+        )
+      }
+      case 4: {
+        return <WriteData bucket={this.state.selectedBucket} />
+      }
+      case 5: {
+        return <ExecuteQuery bucket={this.state.selectedBucket} />
+      }
+      case 6: {
+        return <ExecuteAggregateQuery bucket={this.state.selectedBucket} />
+      }
+      case 7: {
+        return (
+          <Finish
+            wizardEventName="cliWizard"
+            markStepAsCompleted={this.handleMarkStepAsCompleted}
+            finishStepCompleted={this.state.finishStepCompleted}
+            finalFeedback={this.state.finalFeedback}
+            setFinalFeedback={this.setFinalFeedback}
+          />
+        )
+      }
+      default: {
+        return <Overview wizard="cliWizard" />
+      }
+    }
+  }
+
+  render() {
+    const {currentStep} = this.state
+
+    return (
+      <Page>
+        <Page.Header fullWidth={false}>
+          {/* Need an empty div so the upgrade button aligns to the right. (Because clockface uses space-between to justifyContent)*/}
+          <div />
+          <RateLimitAlert location="firstMile.homepage" />
```

Comment:
does this `location` property need to change to something referncing the cli wizard?